### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jburel**
Remove unsupported version and fix renaming of files

### CODE REVIEW

**Comment by jburel** [comment]
required changes from https://github.com/ome/omero-install/pull/154 to avoid travis failure

File: linux/test/README.md [hunk 1]
```diff
@@ -175,21 +175,13 @@ For example:
...
 
 The supported values are: 
-OMERO-DEV-latest, OMERO-DEV-merge-build, 5.0 (latest i.e. 5.0.8), 5.1 (latest), 5.2 (latest), latest (default)
-
-If you select 5.0, you will have to apply a Java security patch
-see http://blog.openmicroscopy.org/tech-issues/2015/07/21/java-issue/
-for information and instructions.
-
-5.1 and 5.0 only work with nginx.
+OMERO-DEV-latest, OMERO-DEV-merge-build,s 5.2 (latest), latest (default)
```
Comment (dominikl) [inline linux/test/README.md]: I guess the 's' just slipped in as typo? But still I don't quite understand, 5.2 is not latest.

**Comment by jburel** [comment]
@dominikl I have removed ref to 5.2 since we no longer do a check

**Comment by dominikl** [comment]
👍  Looks good, ready to merge from my point of view.

**Pull request merged**